base/floatfuncs.jl

```diff
@@ -44,43 +44,51 @@ isinteger(x::AbstractFloat) = (x - trunc(x) == 0)
 
 """
     round([T,] x, [r::RoundingMode])
-    round(x, [digits; base = 10])
+    round(x, [r::RoundingMode]; digits::Integer= [, base = 10])
```

Comment:
`digits::Integer=0, base::Integer=10`?

Reply:
I thought about that, but the default `digits = 0` is covered by the above.

Reply:
Then just omit `=`. Though I'm not sure it's a problem to repeat the default. Also brackets are never used for keyword arguments.

BTW, below there's a typo in "provied".